aten/src/ATen/native/transformers/cuda/mem_eff_attention/gemm_kernel_utils.h

```diff
@@ -207,7 +181,7 @@ struct call_conditional<false, TA, TB> {
 ////////////////////////////////////////////////////////////////////////////////
 
 CUTLASS_DEVICE int32_t warp_uniform(int32_t value) {
-  return (int32_t)__shfl_sync(0xfffff, (unsigned)value, 0);
+  return (int32_t)__shfl_sync(0xffffffff, (unsigned)value, 0);
```

Comment:
dumb q why is it bigger? https://godbolt.org/z/Gbhjojevv
was this wrong before?

Reply:
This wasn't correct before, but without consequences.
This function takes a value from thread 0, and broadcasts it to all the other 32 threads in the mask. Before, it was broadcasting it only to a subset of the thread.
In practice, it does not matter as all thread *already* have the right value, as this is just used to signal to the compiler/optimized that a value is uniform across a warp, and it can save registers.